addons/addon-environment-sc-api/packages/environment-type-mgmt-services/lib/environment-type/env-type-candidate-service.js

```diff
@@ -255,5 +270,25 @@ class EnvTypeCandidateService extends Service {
       ...args,
     );
   }
+
+  /**
+   * A private utility method to find a portfolio id from the department store table
+   * @returns {Promise<string>}
+   */
+  async getPortfolioId() {
```

Comment:
Add a test?

Reply:
Incorporated the suggestion and updated the PR